ios/Mappers.swift

```diff
@@ -3,15 +3,15 @@ import StripeTerminal
 class Mappers {
     class func mapFromReaders(_ readers: [Reader]) -> [NSDictionary] {
         var readersList: [NSDictionary] = []
-
+        
```

Comment:
was this from Xcode re-indenting? for the native repo we run with the rm all whitespace options set and would be good to stay in sync there so we don't thrash 🤷‍♂️ 

in the "while editing" prefs:
<img width="441" alt="Screenshot 2024-05-17 at 11 33 48" src="https://github.com/stripe/stripe-terminal-react-native/assets/62762265/514e454d-be38-4300-9193-4a10f79de622">